### ISSUE TITLE
Add files Copy Bundle Resources 

### ISSUE THREAD

**Issue by KarthikS941** [kronenthaler/mod-pbxproj#171]
When i add a folder using add_folder. All its files are not listed under 'Build Phases' -> Copy Bundle Resources. 

How do it add a file to 'Copy Bundle Resources'.

**Comment by kronenthaler** [comment]
If you are adding a folder as a reference (create_groups=False), only the root folder will be added to the Copy Bundle Resources phase.
If you are adding the folder as a group, then only the files detected as resources will by added under the Copy Bundle Resources phase. Files detected as code (.m, .mm), as headers, or other types will be added to the corresponding phases (compile, headers, etc)

I would need a more clear example of the nature of your folder structure and files contained in it in order to investigate further.

**Comment by kronenthaler** [comment]
Due to the lack of feedback and activity i consider the answer solved the problem.

**Comment by sewonist** [comment]
Hi All

I have two bundles out side project.  Project's hierarchy is like below.

<img width="327" alt="image" src="https://user-images.githubusercontent.com/352759/198519566-a692ac17-bdaa-4ae7-a226-a67ce1865fb5.png">

I just add them using `add_folder` like below.

```
project.add_folder('resources/LUTs.bundle')
project.add_folder('resources/Stickers.bundle')
```

At the time, The bundles was added like below it. and project build has failed.

<img width="610" alt="image" src="https://user-images.githubusercontent.com/352759/198520452-f3e4f184-acae-4a1b-a02b-38aa288c474b.png">

I want add like below image.

<img width="489" alt="image" src="https://user-images.githubusercontent.com/352759/198520589-7512a2aa-8aee-4a12-86b8-4ff66dc75c99.png">

Please let me know any idea for fix this issue.
Thanks,

**Comment by kronenthaler** [comment]
Have you tried what it's suggested in the [answer](https://github.com/kronenthaler/mod-pbxproj/issues/171#issuecomment-340250762)?

TL;DR: add `create_groups=False` to the add_folder function call.
